text/0001-poet2-consensus.md

```diff
@@ -0,0 +1,109 @@
+- Feature Name: PoET-2.0
+- Start Date: 2018-05-21
+- RFC PR: (leave this empty)
+- Sawtooth Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+As blockchain technology matures, there is a pressing need for more efficient consensus mechanisms that will scale to large networks without putting potentially crippling demands on the infrastructure required to support it. Proof of Elapsed Time (PoET) is an attempt at providing an innovative algorithm that not only scales very efficiently, but also places very few demands on the infrastructure.
+This document details a new mechanism for the PoET algorithm that overcomes some of the challenges with the original algorithm.
+The intended audience of this document is architects, developers and anyone who would like to get a better understanding of the details of the PoET 2.0 algorithm.
+
+This RFC describes a new PoET 2.0 Consensus for supporting the SGX PSE-free machines.
```

Comment:
And a minor note:  Best tech-writing practice is to use the full term "Intel® Software Guard Extensions (SGX)" here, because it's the first time the term is used.  But if that makes the sentence confusing or too long, it's probably okay to use "SGX" here, because the full term is in the next section.